cmd/up/up.go

```diff
@@ -321,6 +321,11 @@ func Up() *cobra.Command {
 				upOptions.Deploy = true
 			}
 
+			if up.Manifest.Type == model.OktetoManifestType && !up.Manifest.IsV2 {
+				oktetoLog.Warning("okteto manifest v1 is deprecated and will be removed in okteto 3.0")
+				oktetoLog.Println(fmt.Sprintf("    %s", oktetoLog.BlueString("Consider upgrading to the new manifest schema: https://www.okteto.com/docs/reference/manifest-migration/")))
```

Comment:
I think we need stronger language since people won't have an option soonish.

Reply:
What about?
```
Follow this guide to upgrade to the new okteto manifest schema:
https://www.okteto.com/docs/reference/manifest-migration/
```